boards/BMS/Src/App/App_CellVoltages.c

```diff
@@ -0,0 +1,129 @@
+#include <stddef.h>
+#include "App_CellVoltages.h"
+
+#include "configs/App_CellConfigs.h"
+
+struct CellVoltages
+{
+    uint16_t *measured_cells;
+    uint32_t  total_num_of_cells;
+    uint32_t  num_of_cells_per_segment;
```

Comment:
I've notices in some places you use size_t some places you use uint32_t, why not here as well then?

Reply:
This should be fixed now